src/util.c

```diff
@@ -582,35 +582,36 @@ int dcc_remove_if_exists(const char *fname)
 
 int dcc_which(const char *command, char **out)
```

Comment:
Can you add a comment about the purpose of this function, which is essentially: 

> search for _command_ in every colon-separated directory inside _PATH_...

except with some special behavior if the generated path contains "distcc"? 

I don't know or remember what that's for -- git blame might show you -- but I guess it's about avoiding loops when distcc itself is on the path?